content/en/docs/best-practices/channel-naming.md

```diff
@@ -106,13 +106,14 @@ Channel names are used to imply what form of upgrade you want to offer for your
 
 The names you choose are notional and up to you to decide, however, picking good channel names requires some basic guidance.  What is described below are different channel naming conventions that are commonly used by the operator community to denote different operator upgrade use cases.
 
-### Naming Convention Rules
+### Naming Conventions
 
 * Channel names are chosen by operator authors as they see fit to meet their upgrade strategies.
 * Channel names are unique to your operator and do not collide with channel names used by other operator providers.
 * Seldom is there a situation where your channel names need to contain information about the Kubernetes or Openshift cluster version they run on.  Only in the case where your operator versions have a dependency on the Kubernetes/Openshift version would you include the cluster version in your channel name.
-* You typically would not include product names in your channels since the channels are unique to your product and will not collide with other channel names used by other operators.
+* You typically would not include product names in your channels since the channels are unique to your product and will not collide with other channel names used by other operators. 
```

Comment:
Minor nit, and probably doesn't matter in the context of the website once it gets renders out, can we remove the trailing space that was added here?

Reply:
fixed in https://github.com/operator-framework/olm-docs/pull/139/commits/8090935ca7aed6a4c5191fc87b830302da32d36e